### PR TITLE
Mission progress

### PR DESCRIPTION
**Pull request by tddumont**
This PR contains the Missions Progress component of the Progress Overview page, a header that allows the user to toggle between Missions Progress and Mastery Progress displays, and a minor fix to the task rubric button.

### CODE REVIEW

File: src/Components/MissionProgress/MissionProgress.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        return data.missions.map((mission) => {
+            return (
+            <div style={{"padding-bottom":"10em"}} onClick={() => changeFocusedMission(mission)}>
+                <CircularProgress style={{transform: "scale(5, 5) translate(20%, 0%) ", color: "#E0E0E0"}} variant="determinate" value={100} /> 
+                {/* saving this for when real progress data can be used
+                <CircularProgress style={{transform: "scale(-5, 5)", color: "#4274F3" }} variant="determinate" value={calculateMissionProgress(progressData.getAllMissionProgress[1].mission.id, progressData)} />  */}
+                <CircularProgress style={{transform: "scale(5, 5) rotate(-90deg)", color: "#4274F3" }} variant="determinate" value={65} /> 
+                <h1 style={{color: "black", fontFamily: "\"Poppins\", sans-serif",  transform: "translate(20%, -90%)", fontSize: "16px", width: "70px", height: "40px" }}>{mission.name}</h1>
+            </div>
+            )
```
Comment (ConnorBoulais) [inline src/Components/MissionProgress/MissionProgress.js:79]: Nice job on circular progress, that turned out really good

File: src/Components/MissionProgress/MissionProgressDetails.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            </TableCell>
+            <TableCell align="left">
+                <ProgressBar
+                    width='400'
+                    height='30'
+                    doneColor={barColor}
+                    leftColor='#CCCCCC'
+                    total={100}
+                    done={score ? score : 0}
+                />
```
Comment (ConnorBoulais) [inline src/Components/MissionProgress/MissionProgressDetails.js:61]: This progress bar turned out looking really good as well

File: src/Components/MissionProgress/MissionProgress.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    //Used to select which Mission Progress information should be shared
+    function changeFocusedMission(mission){
+        setFocusedMission({
+            id: mission.id,
+            name: mission.name,
+            description: mission.description
+        })
+    }
+    
+    //Display all missions
```
Comment (ryanmadamba) [inline src/Components/MissionProgress/MissionProgress.js:68]: Good way of displaying the circular progress bar

File: src/Components/ProgressOverview.js [hunk 4]
```diff
@@ -1,284 +1,33 @@
...
-    return(
-      <div className="card">
-        <h1>No data found!</h1>
-      </div>
+  function ProgressTabs() {
+    return (
+        <div className="tabButtonContainer">
+          <button className={activeTab==1 ? "tabButton" : "tabButtonNotActive" } onClick={()=>setActiveTab(1)}>Missions</button>
+          <button className={activeTab==2 ? "tabButton" : "tabButtonNotActive" } onClick={()=>setActiveTab(2)}>Learning Targets</button>
+        </div>
```
Comment (ConnorBoulais) [inline src/Components/ProgressOverview.js:15]: Tab bar looks exactly like design

**Review by ConnorBoulais — APPROVED** [review]
Awesome job!  This came out looking really good

File: src/Components/NavDrawer.js [hunk 5]
```diff
@@ -76,7 +71,7 @@ function NavDrawer(props) {
 
           <ListItem button key="Progress Overview" onClick={() => (props.history.push('/progoverview'))}>
             <ListItemIcon><SpellcheckIcon /></ListItemIcon>
-            <ListItemText className="buttonText" primary="Progress Overview" />
```
Comment (ryanmadamba) [inline src/Components/NavDrawer.js:79]: "Progress" is better than calling it "Progress Overview"

File: src/Components/Task.css [hunk 6]
```diff
@@ -202,16 +202,17 @@
   text-align: center;
   width: 150px;
   height: 50px;
-  margin-right: 100px;  
+  margin-right: 0;  
   margin-top: 75px;
-  margin-left: 1800px;
+  transform: rotate(270deg) translate(0%, 70%);
```
Comment (ryanmadamba) [inline src/Components/Task.css:207]: Good job fixing the Task Rubric Button

**Review by ryanmadamba — APPROVED** [review]
Looks Great, Ted! I like how much you comment your code.

**Pull request merged**